packages/core/src/internal/clients/cache/IDataClient.ts

```diff
@@ -239,6 +241,16 @@ export interface IDataClient {
     sortedSetName: string,
     values: string[] | Uint8Array[]
   ): Promise<CacheSortedSetRemoveElement.Response>;
+  sortedSetLength(
+    cacheName: string,
+    sortedSetName: string
+  ): Promise<CacheSortedSetLength.Response>;
+  sortedSetLengthByScore(
```

Comment:
This is more of a comment on our protos than anything in this pr, but it seems like these could have easily been a single method with optional params? Cause if you call `sortedSetLengthByScore` and don't pass in a min or max value, its the same as just called `sortedSetLength`

Reply:
With our `listFetch` api, we pass in optional indexes, similar to our `sortedSetLengthByScore` here.

```
  listFetch(
    cacheName: string,
    listName: string,
    startIndex?: number,
    endIndex?: number
  ): Promise<CacheListFetch.Response>;
```

Reply:
That's a good point. Should I collapse it into just one new endpoint `sortedSetLength` with the optional start and end score parameters?

Reply:
Well, I guess for consistency with the [Go SDK](https://github.com/momentohq/client-sdk-go/pull/339), it should probably remain as two endpoints.

Reply:
ah yeah if its already in the go sdk then yea probably good to keep it consistent

Reply:
cc @malandis @pratik151192

Reply:
That's true @bruuuuuuuce; but we designed the ```fetch``` equivalent similarly as well. So it was too late to revert to clubbing under the same API.